### PR TITLE
instructions for using git proxies

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@adellape ptal.

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -134,7 +134,29 @@ of a missing `save-artifacts` script.
 For information on how to create a builder image supporting incremental builds
 see the link:../creating_images/s2i.html[creating images guide].
 
-[[starting-a-build]]
```
Comment (adellape) [inline dev_guide/builds.adoc]: Anchor for the next section got clobbered here.

**Comment by adellape** [comment]
Just the one comment, otherwise LGTM.

**Comment by bparees** [comment]
@adellape de-nitted, thanks for the review+catch.

**Comment by adellape** [comment]
:+1:

**Comment by adellape** [comment]
@bparees Hmm, per https://bugzilla.redhat.com/show_bug.cgi?id=1235129, I suppose this should be conditionalized to only appear in the Origin docs until it lands in OSE, correct?

**Comment by bparees** [comment]
yeah i guess so, sorry about that.

**Pull request merged**